README.md

```diff
@@ -1 +1,360 @@
+
+Steps: 
+1. Generate barcode in Sylo 
+2. Create a request in requests.create, using fields: 
+3. Amount 
+4. Asset 
+5. MerchantID 
+6. PatronCode (Enter barcode) 
+
+>Once you have created the request, the apk should display payment option/ confirmation. 
+
+Example:
+
+An example URL that a **POST** request could be sent to would be the following:
+
+``` 
+https://service.centrapay.com/payments/api/requests.create?amount=100&asset=NZD&merchantId=1399b053-b3dd-4c5b-9859-b5bf5c2ac477&patronCode=2368720114399939″-H“accept:application/json”-H“x-api-key:1399b053-b3dd-4c5b-9859-b5bf5c2ac477
+```
+
+#### Non integrated
+
+Where the merchant is unable to integrate electronically, Centrapay can issue the merchant with a unique QR code which can be used at the point of sale to engage consumers and receive payment.
+
+
+## Im not sure about this section
+
+### [Creating a payment request](https://service.centrapay.com/payments/api/documentation#/Requests/postRequestscreate)
+
+
+A payment request can be created by sending a **POST** request to the following endpoint:<br>
+https://service.centrapay.com/payments/api/requests.create
+
+The following parameters **must** be provided with the request:
+
+| Parameter  | Description                                 |
+|------------|---------------------------------------------|
+| amount     | The payment amount in cents                 |
+| asset      | The currency - NZD or AUD                   |
+| merchantId | The ID of the merchant creating the request |
+
+<br>
+
+The following parameters can **optionally** be provided with the request:
+
+| Parameter         | Description                                                  |
+|-------------------|--------------------------------------------------------------|
+| clientId          | The ID of the merchant specific client configuration         |
+| description       | Description of the payment                                   |
+| externalReference | Unique merchant reference for the payment request            |
+| notifyUrl         | The URL that will receive **POST** requests from the webhook |
+
+An example URL that a **POST** request could be sent to would be the following:
+
+```
+https://service.centrapay.com/payments/api/requests.create?amount=100&asset=NZD&description=Test%20purchase&merchantId=1399b053-b3dd-4c5b-9859-b5bf5c2ac477&notifyUrl=https%3A%2F%2Fexample.com%2Fservice.notify
+```
+
+<br>
+
+### [Paying a payment request - Pocket Voucher](https://service.centrapay.com/payments/api/documentation#/requests/postRequestspay)
+
+#### Step 1. Test Voucher Allocation
+
+To generate a test value voucher text ‘CENTRALBONUS’ 393. <br>
+You will then receive a response text containing an 8 digit voucher code that has $20 of loaded credit. 
+
+>Note:  
+The received code is only valid for two weeks from the issue date.<br>
+You might get charged your standard text rates from your provider.
```

Comment:
I think Step 1 and Step 2 is bad because if someone already has a bunch of vouchers they don't need to go through that step. So maybe we just have it as a note on how to generate test vouchers and then explain the redemption process below